File: Python/PRAK301-2210817210013-Muhammad-Shandy-Dailami.py
for x in range(3):
    angka1, angka2, angka3 = input("").split()
    angka1 = int(angka1)
    angka2 = int(angka2)
    angka3 = int(angka3)

    if angka1 < angka2 and angka1 < angka3 and angka2 < angka3:
        print(angka1, angka2, angka3)
    elif angka1 < angka2 and angka1 < angka3 and angka2 > angka3:
        print(angka1, angka3, angka2)
    elif angka2 < angka1 and angka2 < angka3 and angka1 < angka3:
        print(angka2, angka1, angka3)
    elif angka2 < angka1 and angka2 < angka3 and angka1 > angka3:
        print(angka2, angka3, angka1)
    elif angka3 < angka1 and angka3 < angka2 and angka1 < angka2:
        print(angka3, angka1, angka2)
    else:
        print(angka3, angka2, angka1)

File: Bahasa C/PRAK301-2210817210013-Muhammad-Shandy-Dailami.c
#include <stdio.h>

int main() {
    int angka1, angka2, angka3, x = 0;
    for(x = 0; x < 3; x++){
        scanf("%d %d %d", &angka1, &angka2, &angka3);

        if(angka1 < angka2 && angka1 < angka3 && angka2 < angka3) {
            printf("%d %d %d", angka1, angka2, angka3);
        }
        else if(angka1 > angka2 && angka1 > angka3 && angka2 < angka3) {
            printf("%d %d %d", angka2, angka3, angka1);
        }
        else if(angka2 > angka1 && angka2 > angka3 && angka1 < angka3) {
            printf("%d %d %d", angka1, angka3, angka2);
        }
        else if(angka3 > angka1 && angka3 > angka2 && angka2 > angka1) {
            printf("%d %d %d", angka3, angka2, angka1);
        }
        else if(angka2 < angka1 && angka2 < angka3 && angka1 < angka3) {
            printf("%d %d %d", angka2, angka1, angka3);
        }
        else if(angka3 < angka1 && angka3 < angka2 && angka1 < angka2) {
            printf("%d %d %d", angka3, angka1, angka2);
        }
        else {
            printf("%d %d %d", angka3, angka2, angka1);
        }
    }

    return 0;
}